changelog.txt

```diff
@@ -10,12 +10,12 @@
 * Dev - Introduce new parameter to WC_Subscription::get_last_order() to enable filtering out orders with specific statuses.
 * Update - Schedule subscription-related events with a priority of 1 to allow for earlier execution within the Action Scheduler.
 * Fix - Ensure admin notices are displayed after performing bulk actions on subscriptions when HPOS is enabled.
+* Fix - Resolved an error when purchasing subscription products on the block checkout with a limited recurring coupon.
```

Comment:
This changelog was erroneously placed in the 7.3.0 entries after 7.3 had been tagged. This change was actually included in 7.4.0 based on https://github.com/Automattic/woocommerce-subscriptions-core/pull/642 being merged on Aug 12th 2024 and [7.4.0 being tagged](https://github.com/Automattic/woocommerce-subscriptions-core/releases/tag/7.4.0)on Aug 17th.